### PR TITLE
sunset and adjust the cron time for OSD with OCP e2e testing

### PR DESCRIPTION
**Pull request by yasun1**
[OCM-9677](https://issues.redhat.com//browse/OCM-9677)

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml [hunk 1]
```diff
@@ -955,8 +955,8 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-workers-marketplace
-- as: aws-ocm-osd-ccs-f7
-  cron: 2 12 2,9,16,23 * *
+- as: aws-ocm-osd-ccs-f14
+  cron: 36 19 14,29 * *
```
Comment (lento-sun) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:959]: Setting the 29th in crontab might not be executed in February.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable.yaml [hunk 2]
```diff
@@ -173,8 +173,8 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-usertags-custom-security-groups
-- as: aws-ocm-osd-ccs-f7
-  cron: 5 2 1,8,15,24 * *
+- as: aws-ocm-osd-ccs-f14
+  cron: 13 6 14,29 * *
```
Comment (lento-sun) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable.yaml:177]: Setting the 29th in crontab might not be executed in February.

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml [hunk 3]
```diff
@@ -11830,7 +11830,7 @@ periodics:
         secretName: result-aggregator
 - agent: kubernetes
   cluster: build05
-  cron: 2 12 2,9,16,23 * *
+  cron: 36 19 14,29 * *
```
Comment (lento-sun) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml:11833]: Setting the 29th in crontab might not be executed in February.

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16-periodics.yaml [hunk 4]
```diff
@@ -39359,7 +39359,7 @@ periodics:
         secretName: result-aggregator
 - agent: kubernetes
   cluster: build05
-  cron: 5 2 1,8,15,24 * *
+  cron: 13 6 14,29 * *
```
Comment (lento-sun) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16-periodics.yaml:39362]: Setting the 29th in crontab might not be executed in February.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml (hunk 1 above)
Comment (yasun1) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:959]: Good catch.

**Comment by yasun1** [comment]
@liangxia Please help review the PR. Thanks.

**Pull request merged**